app/src/main/java/com/x8bit/bitwarden/ui/platform/manager/intent/IntentManagerImpl.kt

```diff
@@ -51,6 +51,20 @@ private const val TEMP_CAMERA_IMAGE_DIR: String = "camera_temp"
  */
 const val EXTRA_KEY_USER_ID: String = "user_id"
 
+/**
+ * Key for the credential id included in FIDO 2 provider "get entries".
+ *
+ * @see IntentManager.createFido2GetCredentialPendingIntent
+ */
+private const val EXTRA_KEY_CREDENTIAL_ID: String = "credential_id"
+
+/**
+ * Key for the cipher id included in FIDO 2 provider "get entries".
+ *
+ * @see IntentManager.createFido2GetCredentialPendingIntent
+ */
+private const val EXTRA_KEY_CIPHER_ID: String = "cipher_id"
```

Comment:
Both of these will need to be public so they can be extracted when we lookup the cipher for authentication.